core/backend/src/ECSchemaXmlContext.ts

```diff
@@ -17,31 +17,52 @@ export type SchemaKey = IModelJsNative.ECSchemaXmlContext.SchemaKey;
 /** @internal */
 export type SchemaMatchType = IModelJsNative.ECSchemaXmlContext.SchemaMatchType;
 
-/** @internal */
+/** Context for deserializing ECSchemas
+ * @beta
+ */
 export class ECSchemaXmlContext {
   private _nativeContext: IModelJsNative.ECSchemaXmlContext | undefined;
 
   constructor() {
     this._nativeContext = new IModelHost.platform.ECSchemaXmlContext();
   }
 
+  /** @internal */
   public get nativeContext(): IModelJsNative.ECSchemaXmlContext {
     assert(undefined !== this._nativeContext);
     return this._nativeContext;
   }
 
+  /**
+   * Adds a file path that should be used to search for a matching schema name
```

Comment:
Please specify in the docs the order in which paths are searched if I call addSchemaPath multiple times. Same for schema locators.